### PR TITLE
Featue/glare card

### PR DESCRIPTION
**Pull request by sivareddyuppathi**

### CODE REVIEW

File: components/content/inspira/ui/glare-card/GlareCard.vue [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { cn } from "~/lib/utils";
+
+interface GlareCardProps {
+  class?: string;
+  cardWidth?: number;
```
Comment (rahul-vashishtha) [inline components/content/inspira/ui/glare-card/GlareCard.vue:39]: I think this prop is not required here, as the width can be set using tailwindcss classes.

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]

File: components/content/inspira/ui/glare-card/GlareCard.vue (hunk 1 above)
Comment (sivareddyuppathi) [reply components/content/inspira/ui/glare-card/GlareCard.vue]: Done

**Comment by rahul-vashishtha** [comment]
@sivareddyuppathi The format check is getting failed in pipeline. Please fix that using `pnpm format:fix`

**Review by rahul-vashishtha — CHANGES_REQUESTED** [review]
Fix lint issues using `pnpm lint:fix` also.

**Comment by sivareddyuppathi** [comment]
@rahul-vashishtha after I ran above 2 comands still it is not showing any changes in the file where it is failed `content/2.components/glare-card.md`. I mean where ever it is showing error in the checks. After running of above 2 commands it is not showing any changes in file.

Can you please check once from your side.

**Pull request merged**